app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:sessions][:email].downcase
+    if user&.authenticate params[:sessions][:password]
```

Comment:
`&.` ở đây có ý nghĩa gì e?

Reply:
&. chỗ này để đơn giản hóa toán tử && và dễ debug hơn vì khi một trong 2 điều kiện sai nó sẽ lập tức trả exception chứ ko trả về false như && :D